.github/workflows/build-native.yml

```diff
@@ -215,21 +231,48 @@ jobs:
               ./ide/provisioning/install-android-tools.sh SDK
             fi
             ./ide/provisioning/install-debian-packages.sh CLEAN
-          elif [ ${{ matrix.os }} = 'macos-12' ]; then
-            brew install \
-              make \
-              quilt \
-              librsvg \
-              imagemagick gettext sox \
-              fmt \
-              sdl2 \
-              libsodium \
-              freetype \
-              libpng libjpeg-turbo \
-              libtiff libgeotiff proj \
-              c-ares \
-              curl \
-              lua
+          elif [ ${{ matrix.os }} = 'macos-12' ] ; then
+            if [ ${{ matrix.target }} = 'OSX64' ]; then
+              brew install \
+                make \
+                quilt \
+                librsvg \
+                imagemagick gettext sox \
+                fmt \
+                sdl2 \
+                libsodium \
+                freetype \
+                libpng libjpeg-turbo \
+                libtiff libgeotiff proj \
```

Comment:
Please create a shell script for osx macos in ide/provisioning/ analog install-debian-packages.sh

The dependency management should be not in github actions, so we depend as little as possible on it.

Reply:
Perfect thx.